autopush/main.py

```diff
@@ -255,11 +261,23 @@ def make_settings(args, **kwargs):
             router_conf["apns"] = {"sandbox": args.apns_sandbox,
                                    "cert_file": args.apns_cert_file,
                                    "key_file": args.apns_key_file}
-        if args.gcm_apikey is not None:
+        if args.gcm_enabled:
+            # Create a common gcmclient
+            list = json.loads(args.senderid_list)
+            senderIDs = SenderIDs(dict(
+                s3_bucket=args.s3_bucket,
+                senderid_expry=args.senderid_expry,
+                use_s3=args.s3_bucket.lower() != "none",
+                senderid_list=list))
+            senderID = senderIDs.getID()
```

Comment:
nit: comment that this is run here just to ensure it works, as the value is tossed.